taskana-adapter-camunda-outbox-rest-spring-boot/src/main/java/pro/taskana/adapter/camunda/outbox/rest/springboot/config/AppConfig.java

```diff
@@ -0,0 +1,15 @@
+package pro.taskana.adapter.camunda.outbox.rest.springboot.config;
```

Comment:
The classname AppConfig should be more specific to identify that this is the config of the outbox-rest-spring-boot app